modules/call_center/call_center.c

```diff
@@ -872,8 +872,9 @@ static int w_agent_login(struct sip_msg *req, char *agent_v, char *state_v)
 
 
 	/* get state */
+	unsigned int gparam_int_value_flag = GPARAM_INT_VALUE_FLAG;
 	if (fixup_get_isvalue( req, (gparam_p)state_v, &state, &agent_s,
-	&flags)!=0 || ((flags|GPARAM_INT_VALUE_FLAG)==0) ) {
+	&flags)!=0 || ((flags|gparam_int_value_flag)==0) ) {
```

Comment:
what difference this change made?

Reply:
GPARAM_INT_VALUE_FLAG is #defined as follows:
#define GPARAM_INT_VALUE_FLAG	(1U<<0)

In the original code, the bitwise comparison always evaluates to false (or 0). By storing the operation in a variable and using it in the compare, yields the correct output and the warning is subdued. 

With the original code that expression in the bitwise OR statement would always evaluate to false, which is not what was intended by that code.  Since this change is not related to the cluster broadcast issue, we can remove it.